xds/internal/test/xds_server_integration_test.go

```diff
@@ -374,6 +375,11 @@ func (s) TestServerSideXDS_SecurityConfigChange(t *testing.T) {
 // (NonForwardingAction), and the RPC's matching those routes should proceed as
 // normal.
 func (s) TestServerSideXDS_RouteConfiguration(t *testing.T) {
+	oldRBAC := env.RBACSupport
+	env.RBACSupport = true
```

Comment:
Every test that doesn't have this tests the no RBAC behavior, and that RPC's proceed as normal (some even with route configuration)?

Reply:
> Every test that doesn't have this tests the no RBAC behavior, and that RPC's proceed as normal (some even with route configuration)?

Does this mean that setting the environment variable will cause them to fail?

If so, that's fine, but it would be nicer if they disabled RBAC manually if needed so that tests will continue to pass when we invert the default.  If not, we should add tests that make sure RBAC-toggled-off is working.

Reply:
All these tests passed before I had this hardcoded check, as they had incoming RPC's match to a VH (on authority  - "*" matches anything) and matched to a route that always matched, prefix. Now, the route configuration in the tests is simply ignored, and the RPC's still proceed as normal. The only times I configured a route configuration in tests to match RPC's to possible !NFA routes are in explicit tests for route configuration, which I had to explicitly toggle. This sound good or is this not what you're asking?

Reply:
I think this is the opposite of what I'm asking for.

It seems like you never set `RBACSupport=false`, which means we are never testing to make sure the state of "RBAC is disabled, but the config has RBAC filters".  We should have a test or two for that case.

Reply:
Will add both tests discussed in meeting.